geopandas/tests/test_vectorized.py

```diff
@@ -412,3 +412,14 @@ def test_concat():
 
     assert c.base == {a, b}
     assert len(c) == 5
+
+
+def test_fill():
+    p = Point(1, 2)
+    P2 = P.fill([0, 3], p)
+    assert P2[0].equals(p)
+    assert P2[3].equals(p)
+    with pytest.raises(TypeError) as info:
+        P.fill([1, 2], 123)
```

Comment:
This is failing with a TypeError despite the `pytest.raises` context manager.  I'm not sure why.  I suspect some Cython issue.  Do you have any experience with this @jorisvandenbossche ?

Reply:
Hrm, it looks like this doesn't fail on travis.  This is probably something to do with my local testing environment.

Reply:
Doesn't fail for me either